content/en/docs/concepts/cluster-administration/_index.md

```diff
@@ -45,7 +45,7 @@ Before choosing a guide, here are some considerations:
 
 ## Securing a cluster
 
-* [Certificates](/docs/concepts/cluster-administration/certificates/) describes the steps to generate certificates using different tool chains.
+* [Certificates](/docs/tasks/administer-cluster/certificates/) describes the steps to generate certificates using different tool chains.
```

Comment:
It may make sense to change the `title` of the task page (in the front matter) to something like:
Manage Certificates or Generate (or Create) Certificates.
In addition to `administer-cluster`, there is another "certificate" related task sub-directory: `TLS`.